### PR TITLE
Add "Gir.toml" with default params

### PR DESCRIPTION
**Pull request by EPashkin**

### CODE REVIEW

**Comment by gkoz** [comment]
This is a nice direction but seems unfinished, am I correct?

File: src/main.rs [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
+    
+    let girs_dir = match args.get_str("-d") {
+        "" => toml.lookup("options.girs_dir")
+                  .unwrap_or_else(|| panic!("No options.girs_dir in config"))
+                  .as_str().unwrap(),
+        a => a
+    };
+    //println!("Dir: {:?}", girs_dir);
+
+    let gir = match args.get_str("<gir>") {
```
Comment (gkoz) [inline src/main.rs]: Calling the library we're interested in currently just "gir" seems confusing, not specific enough.

**Comment by gkoz** [comment]
Don't we want a command line parameter to point to a config file (they might be several of them)?

File: src/main.rs (hunk 1 above)
Comment (EPashkin) [inline src/main.rs]: You called it that, gi_rust (as analog of grust)?

**Comment by EPashkin** [comment]
Yes it only first step.
Currenly loading single gtk30.gir takes 30sec, IMHO too long :(
I disabled loading others for debug purposes (not commit that)

File: src/main.rs (hunk 1 above)
Comment (EPashkin) [inline src/main.rs]: Sorry, misreaded (think about repository name)
I think gir will be universal converter in future.

Comment (EPashkin) [inline src/main.rs]: or may be stay with <lib>

**Comment by EPashkin** [comment]
Not sure about config file location will be necessary. May be in future.

**Comment by gkoz** [comment]
> Currenly loading single gtk30.gir takes 30sec, IMHO too long

The debug build is slow but if you build with `--release` (and run `target/release/gir`) the speed is not bad.

File: src/main.rs (hunk 1 above)
Comment (gkoz) [inline src/main.rs]: The value of this argument is the name of a specific library (Gtk, Gdk, etc) we want to work with.

**Comment by EPashkin** [comment]
Yes, with release it 3sec or (5sec with all libs). Much better.

**Pull request merged**